src/routes/app/+page.svelte

```diff
@@ -44,45 +32,34 @@
   {#await searchResults}
     <Empty icon={Hourglass24}>Loading…</Empty>
   {:then results}
-    <ResultsList {results} />
+    <ResultsList
+      results={results.results}
+      count={results.count}
+      next={results.next}
+      auto
+    />
   {/await}
 
   <PageToolbar slot="footer">
     <label slot="left">
-      <input type="checkbox" name="select_all" />
-      Select all
+      <input
+        type="checkbox"
+        name="select_all"
+        checked={$selected.length === $visible.size}
+        indeterminate={$selected.length > 0 && $selected.length < $visible.size}
+        on:change={selectAll}
+      />
+      {#if $selected.length > 0}
+        {$selected.length.toLocaleString()} selected
+      {:else}
+        Select all
```

Comment:
Just flagging that this should be using i18n, correct? This is something I've been lax about in my code too; if we want to commit to doing an i18n pass closer to launch, I'm fine with that too.

Reply:
It should. There are a bunch of places where we need i18n. I'll open a new ticket to do that. We're going to have a bunch of unused translations as we move components over, so I'm trying to think through a way to do cleanup.

Reply:
I've been thinking there may be some neat LLM tooling to generate localization JSON files, too… A new ticket sounds great.